plugins/main/bridge/bridge.go

```diff
@@ -658,6 +675,13 @@ func cmdDel(args *skel.CmdArgs) error {
 		return err
 	}
 
+	if n.MacSpoofChk {
+		sc := link.NewSpoofChecker("", "", uniqueID(args.ContainerID, args.IfName))
```

Comment:
I understand the current implementation of `sc.Teardown` doesn't need the iface name and the mac address to work and that's why you are passing empty strings.
But it is an internal detail of the `sc.Teardown` implementation, I wouldn't rely on it.

Reply:
Seems the project doesn't have e2e tests at all. So I guess your "integration" test here is the best we can have.